specification/inference/inference/InferenceResponse.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { InferenceResult } from '@inference/_types/Results'
+
+export class Response {
+  body: {
+    predicted_value: InferenceResult
```

Comment:
I wonder if we should include a field in the response that indicates what type it is. OpenAI does this using the `object` field. I wonder if it'd be easier for clients to key off of that field rather than remembering the task type in the request URL 🤷‍♂️

Reply:
@davidkyle for `SparseVector` should we call it `SparseEmbedding` and have the field be `sparse_embedding`?

The `TaskType` in the java code is `SparseEmbedding`.

Reply:
👍 thanks for reviewing @jonathan-buttner. I've renamed the field `sparse_embedding` but kept the type as `SparseVector`. The reasoning being that the mapping types in Elasticsearch are `sparse_vector` and `dense_vector`. A `text_embedding` goes in a `dense_vector` and a `sparse_embedding` in a `sparse_vector`